internal/pkg/routes/route.go

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) Mainflux
+// SPDX-License-Identifier: Apache-2.0
+
+package routes
+
+import (
+	mqtt "github.com/eclipse/paho.mqtt.golang"
+	"github.com/mainflux/mainflux/logger"
+	"github.com/nats-io/nats.go"
+)
+
+type R struct {
```

Comment:
Name of this structure does not tell anything. It is not self-documenting, and it even misses comment to explain what it is.

Please change this.